Cargo.toml

```diff
@@ -93,6 +93,7 @@ once_cell = "1.20.2"
 openssl = { version = "0.10.68" }
 pin-project = "1.1.7"
 pin-project-lite = "0.2.15"
+postcard = { version = "1.1.1", features = ["alloc", "use-std"] }
```

Comment:
library for no_std (de)serialization. made and maintained by a well-known embedded rust developer. https://github.com/jamesmunns/postcard

can be used for custom protocols and wire formats